tasks/9_init.yml

```diff
@@ -9,25 +9,35 @@
   when:         ansible_os_family == "RedHat"
   notify:       restart_seafile
 
-- name:         provision init script
+- name:         provision sysvinit script
   template:
     src:        init/{{ item.name }}.initd.{{ ansible_os_family }}
     dest:       /etc/init.d/{{ item.name }}
     owner:      root
     group:      root
     mode:       0750
-  when:         ansible_os_family in item.os_family
+  when:         ansible_service_mgr == "sysvinit"
```

Comment:
IIRC, when I started out this role (seafile 4.x) that was how the docs explained what needed to be done, and those were different scripts. It is quite possible that this changed over time, and I can't find anything similar any more. seafile.sh and seahub.sh will probably have evolved, too, and shoudl perhaps be updated in the role, or the logic rewritten as that the files deployed from the tarball are used to update them.